cranelift/codegen/src/dominator_tree.rs

```diff
@@ -439,6 +439,13 @@ impl DominatorTreePreorder {
             }
         }
     }
+
+    /// Clear the data structure
+    /// Recursively clears its underlying children
+    pub fn clear(&mut self) {
+        self.nodes.clear();
+        self.stack.clear();
+    }
```

Comment:
Is this function still needed? It doesn't look like it's referenced from the changes in this PR.

Reply:
hi there :) I double checked and it seems nothing is calling this function. I've removed it. Thank you for the review :)